src/context/register-context.js

```diff
@@ -0,0 +1,37 @@
+import { useReducer, createContext, useContext } from "react";
+import { registerReducer } from "../reducers";
```

Comment:
Not mandatory but good to know: You can use a path alias to avoid moving back in folders using ../../
refer For Create React App without using Craco section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160